### PR TITLE
Data Analysis

### PR DESCRIPTION
**Pull request by CrispyBacon1999**

### CODE REVIEW

File: analysis/basics_of_analysis.rst [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+.. figure:: media/matplotlibdemo.png
+
+.. code-block:: python
+
+    '''
+    NOTE: The following code requires a few modules to run.
+                numpy
+                matplotlib
+                frctba
```
Comment (prajy) [inline analysis/basics_of_analysis.rst]: It looks like its just the libraries needed. numpy is math lib. frctba is the library that can pull from the TBA API. matplotlib is some graphing visualization lib.

Comment (Twinters007) [inline analysis/basics_of_analysis.rst]: I know what the first two libraries are, not sure what frctba is

**Comment by TuringsEgo** [comment]
https://pypi.python.org/pypi/frctba/1.4 ?

File: analysis/basics_of_analysis.rst (hunk 1 above)
Comment (TuringsEgo) [inline analysis/basics_of_analysis.rst]: https://pypi.python.org/pypi/frctba/1.4

**Pull request merged**